ksqldb-engine/src/test/java/io/confluent/ksql/function/udaf/array/CollectListUdafTest.java

```diff
@@ -65,14 +66,16 @@ public void shouldMergeIntLists() {
   @Test
   public void shouldRespectSizeLimit() {
     final TableUdaf<Integer, List<Integer>, List<Integer>> udaf = CollectListUdaf.createCollectListInt();
+    ((Configurable) udaf).configure(ImmutableMap.of(CollectListUdaf.LIMIT_CONFIG, 10));
+
     List<Integer> runningList = udaf.initialize();
-    for (int i = 1; i < 2500; i++) {
+    for (int i = 1; i < 25; i++) {
       runningList = udaf.aggregate(i, runningList);
     }
-    assertThat(runningList, hasSize(1000));
+    assertThat(runningList, hasSize(10));
     assertThat(runningList, hasItem(1));
-    assertThat(runningList, hasItem(1000));
-    assertThat(runningList, not(hasItem(1001)));
+    assertThat(runningList, hasItem(10));
+    assertThat(runningList, not(hasItem(11)));
```

Comment:
If limit is 10, why does it have the item 11?

Reply:
😉 the code is `not(hasItem(11))`

Reply:
Pfff. I see it. sorry